### DISCUSSION TITLE
Users meeting

### DISCUSSION THREAD

**Discussion by jhollway** [stocnet/goldfish#22] [Ideas]
Hi all,

I'm Daniel Karell, an assistant professor in the Department of Sociology at Yale University. My research is broadly about discursive communities and extremism. I'm currently using goldfish models to examine the growth of online right-wing groups. My primary dataset comprises timestamped YouTube videos and comments; network ties are comments on videos and comments on other comments.

I'm generally curious about how others are approaching multiplex and multilevel networks. (The published papers have been a great start!) I'm also interested in hearing about how others interpret results, especially with complex models. (I often present to audiences accustomed to seeing results as marginal effects, so I think about interpretation in that sense, too.) And, I'm always keen to discuss ideas about socio-semantic networks!

I think it'd be great to have a "users meeting" (or meetings) to share and comment on projects, ask and discuss general questions, and so on. I'm happy to help organize if there is interest.

I'm looking forward to "meeting" you all!

Originally posted by @dkarell in #20 (comment)

**Comment by jhollway** [comment]
Excellent idea @dkarell! I'm opening this up as a new discussion thread to which other users can, if they wish, post a quick introduction. I'll then add names to a list of those interested in such 'users meetings', which if I understand correctly, are for researchers who are more advanced than the introductory goldfish workshop but not (yet) interested in contributing to method/software development?

**Reply by dkarell** [reply]
Yes, something like that @jhollway. I had in mind a RSiena advanced users meeting I attended a few years ago. There were three or four paper presentations, comments and questions on the presentations from participants, and then Tom gave a talk covering new ideas and directions for RSiena, followed by more discussion. It served to share ideas, refine work using RSiena, and build community.

It's just an idea/suggestion. Something like this would be useful for me since I'm learning/using goldfish in a solitary context on this side of the Atlantic. I obviously haven't attended any introductory workshops (I must have missed those announcements) so I don't know how many people are currently using goldfish. Maybe there isn't a demand for meetings yet...

**Reply by marion-hoffman** [reply]
Hi Daniel and James,
That's a nice idea! I don't know who would be interested but we can always ask around. At least from my part, I would be happy to participate in such a meeting. Maybe it could be coupled with a quick online tutorial/introduction, since workshops are no longer happening these days? 
In any case, happy to help.
Best, Marion

**Comment by jmueller17** [comment]
Hi there, 

I would definitely sign-up for such an opportunity. A while ago I had the chance to build a DyNAM model with the help of Marion and Christoph in Zürich for the interaction patterns in small research teams (based upon wearable sensor data). However, I'm still struggling to build a more complex, hierarchical model for integrating the 11 teams into a single analysis. There is this paper by DuBois [1] for hierarchical models for REMs, but I haven't been able to track down something similar for DyNAM. Any help/discussion in this direction - either during an event or by other means - would be fantastic! 

[1] DuBois, Christopher, Carter T. Butts, Daniel McFarland, and Padhraic Smyth. 2013. “Hierarchical Models for Relational Event Sequences.” Journal of Mathematical Psychology 57 (6): 297–309. https://doi.org/10.1016/j.jmp.2013.04.001.

**Reply by auzaheta** [reply]
Hi @jmueller17, I realized of this thread just today. 
I've been working on an extension of DyNAMs using hierarchical models. Sadly, a software implementation will take some time to be ready. I'm happy to collaborate on implementing it on your data. So far, I'm using `goldfish` for the preprocessing, some plain `r` to shape the data, and `rstan` for the inference.

**Reply by jmueller17** [reply]
Hi @auzaheta, that would be excellent, thank you! I will contact you via email to explain where we stand and how/if we could collaborate.
